distributed/client.py

```diff
@@ -498,6 +498,19 @@ class AllExit(Exception):
     """Custom exception class to exit All(...) early."""
 
 
+def _handle_print(event):
+    _, msg = event
+    if isinstance(msg, dict) and "args" in msg and "kwargs" in msg:
+        print(*msg["args"], **msg["kwargs"])
+    else:
+        print(msg)
```

Comment:
Just for my own understanding, is this logic is to handle the case that users manually calls `worker.log_event("print", ...)`? If so, should we handle the case that a user specifies only one of `args` or `kwargs`?

Reply:
Since we're defining this handler, we could also punt on handling cases other than `print(*msg["args"], **msg["kwargs"])`

Reply:
I prefer one if clause over spurious exceptions and think this won't hurt anyone, however unlikely it is.